src/game.cpp

```diff
@@ -10649,13 +10649,19 @@ void game::mend( int pos )
     }
 }
 
-bool add_or_drop_with_msg( player &u, item &it )
+bool add_or_drop_with_msg( player &u, item &it, const bool unloading = false )
 {
     if( it.made_of( LIQUID ) ) {
         g->consume_liquid( it, 1 );
         return it.charges <= 0;
     }
-    if( !u.can_pickVolume( it ) ) {
+
+    //Try to add ammo to container
```

Comment:
Why this comment? Isn't this exactly what the called function does? Why do you think this needs a comment, especially as you have added the *same* comment to both function calls? If the function name does describe what it does, it should be changed.